scripts/download_combadge_assets.py

```diff
@@ -0,0 +1,22 @@
+import datetime
```

Comment:
Is this file used anywhere?

Reply:
No, it's a useful snippet though. Without it you can't run integration tests